### PR TITLE
chore: set up graphql

### PR DESCRIPTION
**Pull request by mapra99**
This is essentially the result of running the graphql generators as stated here
https://graphql-ruby.org/

It also includes another gem to have a graphical UI to play with the API, at http://localhost:3000/graphiql

### CODE REVIEW

File: api/app/assets/config/manifest.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+//= link graphiql/rails/application.css
+//= link graphiql/rails/application.js
```
Comment (ImMPrada) [inline api/app/assets/config/manifest.js:2]: this is for the js graphql controller, right? even being an API?

**Review by ImMPrada — APPROVED** [review]
o.O questions

File: api/app/assets/config/manifest.js (hunk 1 above)
Comment (mapra99) [reply api/app/assets/config/manifest.js:2]: image
It's kinda like a postman

Comment (ImMPrada) [reply api/app/assets/config/manifest.js:2]: ah! ajjajaja

**Pull request merged**